test/index.js

```diff
@@ -11,6 +11,10 @@ function run(route, url, loose) {
   return out;
 }
 
+function raw(route, url, loose) {
```

Comment:
Two separate functions `exact` and `loose` may improve readability.

Reply:
Maybe another time :) This helped legibility for me now while covering my tracks